osbs/build/build_request.py

```diff
@@ -1464,12 +1464,12 @@ def render_node_selectors(self):
             if self.platform_node_selector:
                 self.template['spec']['nodeSelector'].update(self.platform_node_selector)
 
-    def render(self, validate=True):
+    def render(self, api=None, validate=True):
         if validate:
             self.spec.validate()
 
         self.render_customizations()
-        self.render_name()
+        self.render_name(self.spec.name, self.spec.image_tag, self.spec.platform)
```

Comment:
Would it be simpler to follow if these render methods took the actual value of the `BuildParams`?
```python
self.render_name(self.spec.name.value, self.spec.image_tag.value, self.spec.platform.value)
```

Reply:
Sure, why not.

Fixed.